apache/superset
# [Explore] including auto generated avg metrics in druid

AVG is a more in-depth post aggregation in druid, and I didn't include that in the v0 of adhoc metrics (https://github.com/apache/incubator-superset/pull/4663). However, that PR did exclude auto-generated avg metrics in druid, meaning there's no way to access and avg-based metric in explore currently. This PR whitelists auto generated avg metrics specifically for druid so those calculations can still be accessed.

test plan:
- view a sqla datasource with auto generated avg metrics. verify they do not appear
- view a druid datasource with autogenerated avg metrics. verify they do appear
- run tests

reviewers:
@michellethomas @mistercrunch @williaster @graceguo-supercat
